docs/docs/README.md

```diff
@@ -0,0 +1,581 @@
+# Welcome to WorkFlow Launcher
+
+## Overview
+
+[WorkFlow Launcher (WFL)](https://github.com/broadinstitute/wfl.git) 
+is a workload manager.
```

Comment:
This is a bit confusing if one is not familiar with the workflow launcher.

Reply:
I agree, not sure how could make it more explicit. @tbl3rd ?

Reply:
I don't have time to work on this now. Much improvement is possible.
Be careful not to make it less useful for Broadies while attempting to
make it more appropriate for a general audience.

I doubt WFL will ever be very useful to outsiders. A lot of work will be
necessary before that happens … if ever.